test/src/utils/apm/assertCanPublish.test.ts

```diff
@@ -0,0 +1,51 @@
+import { assert } from 'chai'
+import { assertCanPublish } from '~/src/utils/apm'
+import { getMainnetProvider } from '~/test/test-helpers/providers'
+import { zeroAddress } from '~/src/params'
+import { BuidlerPluginError } from '@nomiclabs/buidler/plugins'
+
+describe('apm > assertCanPublish', () => {
+  const provider = getMainnetProvider()
+  const financeManager = '0xE04cAbcB24e11620Dd62bB99c396E76cEB578914'
+  const testCases: {
+    [appName: string]: {
+      [sender: string]: boolean // expected isAllowed
+    }
+  } = {
+    'finance.aragonpm.eth': {
+      [financeManager]: true,
+      [zeroAddress]: false
+    },
+    'newapp.aragonpm.eth': {
```

Comment:
Does the newapp repo already exist? Otherwise, I wonder how we make sure `financeManager` has the `CREATE_REPO_ROLE`

Reply:
Because I'm checking that in the registry contract `aragonpm.eth` not the app repo itself